lua/nvim-treesitter/textobjects/select.lua

```diff
@@ -48,14 +48,22 @@ function M.attach(bufnr, lang)
   lang = lang or parsers.get_buf_lang(buf)
 
   for mapping, query in pairs(config.keymaps) do
+    local desc
+    if type(query) == "table" then
+      desc = query.desc
+      query = query.query
+    end
+    if not desc then
+      desc = "Select textobject " .. query
+    end
     if not queries.get_query(lang, "textobjects") then
       query = nil
     end
     if query then
       local cmd_o = ":lua require'nvim-treesitter.textobjects.select'.select_textobject('" .. query .. "', 'o')<CR>"
```

Comment:
Same here: we should make this a pure Lua mapping; that's the whole point of this API.